{"uuid": "60e25ef7-7dc8-4697-8e51-3bd140483176", "children": ["33ea8e80-aefc-4189-9aab-3ea92e78b64e", "71960e3a-2005-40c1-9b50-f35d135a5313"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\python.py\", line 703, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"F:\\python\\zrapiautotestv1.0\\TestCase\\StockQuotation\\OptionalStock\\test_stick_optional_stock.py\", line 17, in setup_class\n    cls._id = {\"id\": str(id[1].get('_id'))}\n"}, "start": 1594175500948, "stop": 1594175501016}], "start": 1594175500948, "stop": 1594175501027}